integration-tests/api/__tests__/admin/discount.js

```diff
@@ -1244,6 +1266,80 @@ describe("/admin/discounts", () => {
           )
         })
     })
+
+    it("fails to create a discount if the regions contains an invalid regionId ", async () => {
+      expect.assertions(2)
```

Comment:
todo: this can be removed since now it is outside of the promise and will always reach the `expect`